src/store/Store.js

```diff
@@ -67,7 +67,17 @@ class Store {
       chrome.runtime.sendMessage({
         type: DISPATCH_TYPE,
         payload: data
-      }, ({error, value}) => {
+      }, (resp) => {
+        if (resp === undefined) {
```

Comment:
Just for some extra safety, can we use the `if (typeof resp === 'undefined')` format for the undefined check?